velox/functions/lib/string/StringImpl.h

```diff
@@ -147,6 +147,38 @@ FOLLY_ALWAYS_INLINE int32_t charToCodePoint(const T& inputString) {
   return codePoint;
 }
 
+/// Returns the sequence of Unicode code point of characters in the an input
+/// string.
+template <bool isAscii, typename TOutput, typename TInput>
+std::vector<TOutput> stringToCodePoint(const TInput& inputString) {
```

Comment:
Let's remove the logic to process ascii input from this function. It will simplify this function and make sure more readable overall.